src/features/Classes/ClassDetailPage/index.jsx

```diff
@@ -80,6 +88,10 @@ const ClassDetailPage = () => {
 
   const handleEnrollStudentModal = () => setIsEnrollModalOpen(!isEnrollModalOpen);
 
+  const handleGradebookButton = () => {
+    window.open(`${gradebookUrl}/${classId}`, '_blank', 'noopener,noreferrer');
```

Comment:
Add the /gradebook/ path because in local is not necessary and won't work, in stage and prod this is required and we want to avoid add another configuration